pages/main/faq.md

```diff
@@ -7,7 +7,43 @@ layout: documentation
 
 This page contains some Frequently Asked Questions.
 ## What is AsteroidOS?
-AsteroidOS is an open source Linux distribution that runs on many different smartwatches. It uses [Qt](http://www.qt.io/) and QML to provide the graphical interface.
+AsteroidOS is an open source Linux distribution that runs on many different smartwatches. It uses [Qt](http://www.qt.io/) and QML to provide the graphical interface.\
+One of the main goals of AsteroidOS is to provide users with greater control over their devices and data privacy. The operating system is fully customizable, and users can modify it to meet their specific needs.
+
+## Which features does AsteroidOS provide?
+Currently, AsteroidOS has these features available:
+ - Always-on-Display
+ - Tilt-to-Wake
+ - Palm-to-Sleep
+ - phone notifications
```

Comment:
nit: can we just capitalize all these bullet points ? I don't have a strong grammatical point to make, actually it's probably wrong but it feels more... aesthetic ? to me... idk ?

hint: If you push back, I'll allow it :)

Reply:
As a german, i ahve to force myself to not capitalize 50% of the english i write anyway. So i am all for it. Imho a list item is kind of a sentence.